test/cli/test_cli.py

```diff
@@ -114,6 +117,45 @@ def test_should_walk_through_all_workflows_inside_package_tree(self):
         self.assertEqual('ID_2', res[1].workflow_id)
         self.assertNotEqual('@once', res[1].schedule_interval)
 
+    def test_walk_workflows_in_packages_should_not_walk_when_no_packages_given(self):
+        # when
+        res = walk_workflows_in_packages(TEST_PROJECT_DIR, [])
+
+        # then
+        self.assertEmpty(res)
+
+    def test_walk_workflows_in_packages_should_walk_through_all_workflows_without_transformer(self):
+        # when
+        res = walk_workflows_in_packages(TEST_PROJECT_DIR, ['test_module'])
+
+        # then
+        self.assertEqual([('ID_1', '@once'), ('ID_2', '@daily'), ('ID_3', '@daily'), ('ID_4', '@daily'), ('ID_5', '@daily')],
```

Comment:
Why do we use map here and chekcs only `workflow_id` and `schedule_interval`

Reply:
I'm mapping in order to avoid comparing whole objects. I followed existing `test_should_walk_through_all_workflows_inside_package_tree` test that asserts on ID and schedule interval.